types/config.d.ts

```diff
@@ -151,6 +151,7 @@ declare global {
       export interface Settings extends Required<SharedFlagsSettings> {
         throttling: Required<ThrottlingSettings>;
         screenEmulation: ScreenEmulationSettings;
+        displayStrings: DisplayStrings;
```

Comment:
nit: is this necessary to set manually? It should already be set to this from the base `Required<SharedFlagsSettings>`, I believe...